plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java

```diff
@@ -507,32 +508,45 @@ protected Composite createPropertyExpressionText(final Composite parent) {
 		propertyExpressionTypeElements = new Control[] { propertyExpressionTextLabel, propertyExpressionText,
 				propertyExpressionHelp };// mouse
 		propertyExpressionText.addMouseListener(new MouseListener() {
+		    
```

Comment:
Fix formatting issue.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804